### ISSUE TITLE
Can't take screenshot

### ISSUE THREAD

**Issue by jason-bowen-zheng** [pyglet/pyglet#405]
Follow [this](https://pyglet.readthedocs.io/en/latest/programming_guide/image.html#saving-an-image) guide, and raise an error:
```
File "/home/jason/mcpy/Minecraft/minecraft/entity/player/player.py", line 221, in on_key_press
    pyglet.image.get_buffer_manager().get_color_buffer().save(os.path.join(search_mcpy(), 'screenshot', name))
  File "/usr/lib/python3.9/site-packages/pyglet/image/__init__.py", line 499, in save
    encoder.encode(self, file, filename)
  File "/usr/lib/python3.9/site-packages/pyglet/image/codecs/png.py", line 101, in encode
    data.fromstring(image.get_data(image.format, image.pitch))
AttributeError: 'array.array' object has no attribute 'fromstring'
```
pyglet version 1.5.17

**Comment by benmoran56** [comment]
This is now fixed in pyglet 1.5.18. Please give that a try to confirm.

**Comment by jason-bowen-zheng** [comment]
Yes, The problem has solved. Thanks!
